.github/workflows/macos-build.yml

```diff
@@ -3,41 +3,60 @@ name: Build MacOS
 # TODO: change to 'pull_request' and 'push' 
 # when we are ready to enable this for PRs.
 on:
-  workflow_dispatch:
+  workflow_call:
+    inputs:
+      IS_RELEASE:
+        required: true
+        type: boolean
+        default: false
+      GN_CONFIG:
+        required: false
+        type: string
+        default: false//electron/build/args/testing.gn
+      GN_BUILD_TYPE:
+        required: false
+        type: string
+        default: testing
+      GENERATE_SYMBOLS: 
+        required: false
+        type: boolean
+        default: false
+      UPLOAD_TO_STORAGE: 
```

Comment:
Can this just be a boolean too?

Reply:
Yes! The reason it's a string right now is purely for backwards compatibility while we run GHA and CircleCI next to each other, our upload scripts unfortunately expect a string 😅 Once both Mac and Linux are over, we can swap this to a boolean